lawlibrary/views/home.py

```diff
@@ -9,6 +9,23 @@ class HomePageView(TemplateView):
 
     def get(self, request, *args, **kwargs):
         context = self.get_context_data()
+
+        court_classes = (
+            CourtClass.objects.select_related("courts")
+            .order_by("name", "courts__name")
+            .values("name", "courts__name", "courts__code")
+            .all()
+        )
+
+        grouped_courts = {}
+        for c_c in court_classes:
+            grouped_courts.setdefault(c_c["name"], [])
+            if c_c.get("courts__name") and c_c.get("courts__code"):
+                grouped_courts[c_c["name"]].append(
+                    {"title": c_c["courts__name"], "code": c_c["courts__code"]}
+                )
+
+        context["grouped_courts"] = grouped_courts
         context["recent_judgments"] = Judgment.objects.order_by("-date")[:5]
         context["recent_legislation"] = Legislation.objects.filter(
             metadata_json__stub=False
```

Comment:
If you move the "Collections" block to liiweb, then you'll also need to move the `Taxonomy.get_tree()` below into that view.